src/awakening_stage/gui/InventoryScreen.tscn

```diff
@@ -98,6 +74,7 @@ margin_left = 100.0
 margin_top = 100.0
 margin_right = -465.0
 margin_bottom = -559.0
+AutomaticWindowReorderingDepth = 0
```

Comment:
Is there a reason why this window is so special that it needs a non-default value here?

(I'm trying to work out all the smaller file's problems to get them out of the way).

Reply:
To me it makes sense for it to be always behind its panels. Another approach would be to put the panels as its children, but when I tried to do it like this the sizes of the windows got a little bit messed up.

Reply:
To me the inventory is the more important screen than the complimentary panels.

Reply:
The panels are smaller and they easily get lost behind the inventory after moving it. Also there is this option to hide/show the panels if one doesn't need them. I will change it back if you still think that this is bad.

Reply:
I didn't bring this up before but I though that maybe when opening a window should reorder to the top? So it is less likely that you lose some small windows as when you reopen them they would come on top. Though if that's a bit complicated to add that could be another PR.

For the inventory I think the inventory should at least be on top of the ground panel by default and it should be able to be brought on top of the other panels. It'd be pretty weird to have that one window specifically not be able to go on top as I'm sure someone will notice and eventually submit a bug report.

Reply:
I will revert the change then. I will take a look at it and I will see. I think that it can be done easily with the signal, but the signal should be renamed to something else than Dragged I think. Maybe something like ReorderRequested?

Reply:
Well I think a signal for drag might be useful in the future so it should be kept. Unless I'm misremembering there should already be some way to signal that the node has become visible. Maybe even the godot standard becoming visible signal could be used?

Reply:
Okay. What about making a signal Opened in CustomDialog and emitting it from the OnOpen method? For CanvasItem there seem to be just hide and visibility_changed signals (I mean from relevant signals) so those signals can't be used directly anyway.

Reply:
That's about what I had in mind: using the virtual open method to emit a signal if there wasn't an already existing signal that couldn't be used.

Reply:
I refactored the Opened signal to the base window as that's where the Closed signal is also so it makes sense for a symmetrical operation to also exist there.